migrations/20220512130201-create-cab-tables.js

```diff
@@ -0,0 +1,11 @@
+const { migrate } = require('../server/utils/migrateUtils');
+
+module.exports = {
+  up: queryInterface => migrate(__filename, queryInterface),
+  down: async function(queryInterface) {
+    await queryInterface.dropTable('drivers');
+    await queryInterface.dropTable('cabs');
+    await queryInterface.dropTable('bookings');
+    return Promise.resolve();
```

Comment:
Like this 
```js
return Promise.all[
  queryInterface.dropTable('drivers'),
  queryInterface.dropTable('cabs'),
  queryInterface.dropTable('bookings')
]

```
https://sequelize.org/docs/v6/other-topics/migrations/

You can also throw error like this
```js
down: () => Promise.reject(new Error('error'))
```